thentos-purescript/src/Servant/Simple.purs

```diff
@@ -23,6 +22,22 @@ postUser body headerxthentossession = affjax $ defaultRequest
     , content = Just body
     }
 
+postUserRegister :: forall eff. String -> String -> Affjax eff Foreign
+postUserRegister body headerxthentossession = affjax $ defaultRequest
+    { method = POST
+    , url = "/user/register"
+    , headers = [RequestHeader "content-type" "application/json", RequestHeader "accept" "application/json", RequestHeader "x-thentos-session" headerxthentossession]
```

Comment:
this is generated code, so changing this would imply changing packgae servant-purescript.

> > +    , headers = [RequestHeader "content-type" "application/json", RequestHeader "accept" "application/json", RequestHeader "x-thentos-session" headerxthentossession]
>
> The `headers` lines are too long (here and below).